extensions/cornerstone/src/initWADOImageLoader.js

```diff
@@ -54,20 +54,39 @@ export default function initWADOImageLoader(
       // we should set this flag to false.
       convertFloatPixelDataToInt: false,
     },
-    beforeSend: function(xhr) {
+    beforeSend: function (xhr) {
       const headers = userAuthenticationService.getAuthorizationHeader();
 
       // Request:
       // JPEG-LS Lossless (1.2.840.10008.1.2.4.80) if available, otherwise accept
       // whatever transfer-syntax the origin server provides.
       // For now we use image/jls and image/x-jls because some servers still use the old type
       // http://dicom.nema.org/medical/dicom/current/output/html/part18.html
+
+      //Initialize the accept header content
+      const acceptHeader = ['multipart/related']
+      //Storing the requesting transfer syntax specified in OHIF config
+      const requestTransferSyntaxUID = appConfig.requestTransferSyntaxUID
```

Comment:
This should be the default TSUID, but the actual request should be able to over-ride the requested TSUID.  Why not just allow configuring the default accept header - INCLUDING making it empty?  If it were set as an array of strings, then an empty array wouldn't set it, and would allow the default response to be sent.  Also, it is permissable to request single part types directly, and it would be nice to allow that to be configured using the same mechanism - for example:
dataSource....defaultImageAccept: []   would not pass any accept header (this is required to work on all DICOMweb servers)
dataSource...defaultImageAccept: ['multipart/related; type="image/jphc"', 'multipart/related; type="image/jpeg"] would allow for either HTJ2K or JPEG responses - something that can be very important when fetching existing lossy encoded images.
Then,
dataSource...defaultImageAccept: ['image/jpeg', 'image/jp2'] would return JPEG or JPEG2000 responses.

Reply:
moving this comment in dicomwebdatasource index.js (WIP)